src/item.cpp

```diff
@@ -2237,12 +2268,22 @@ bool item::is_auxiliary_gunmod() const
 
 int item::get_storage() const
 {
-    const auto t = find_armor_data();
-    if( t == nullptr ) {
+    float pockets = 1;
+    auto t = find_armor_data();
+    if( t == nullptr )
         return 0;
-    }
+
     // it_armor::storage is unsigned char
-    return static_cast<int>( static_cast<unsigned int>( t->storage ) );
+    int result = (static_cast<int> (static_cast<unsigned int>( t->storage ) ) );
+
+    if (item::item_tags.count("pocketed") > 0){
+    pockets = 1.5;
+        if (result > 0)
+            return result * pockets;
+        else return result + pockets + 3.5; // Can I just use return 5?
```

Comment:
>Can't be used on FANCY or SUPER_FANCY gear, as the pockets would just undermine the look.

Unless your tailoring is 10+. Then you can pull it off. And just tear off the FANCY and downgrade SUPER_FANCY before that, with a warning prompt.